document license text being usable in kickstart (#2550)
FusionAuth/fusionauth-site: site/docs/v1/tech/installation-guide/kickstart.adoc

To use a variable, define the key and value in the `variables` section of the kickstart file and then access that value anywhere outside of the `variables` section using the `#{foo}` notation where `foo` is the name of the variable.

> Here is the same example as above, but instead of hard coding the API key, the API key value is a variable which can be reused. This example is not particularly interesting since the value has been moved from the `apiKeys` section to the `variables` section. Never worry! Using variables can come in handy as you'll see in the next few examples.

Reviewer: ```suggestion
Here is the same example as above, but instead of hard coding the API key, a variable is created to hold the API key value. This example is not particularly interesting since the value has been moved from the `apiKeys` section to the `variables` section. Never worry! Using variables can come in handy as you'll see in the next few examples.
```

There is a top level property in the request called `tenantId` and you simply set that value to indicate which Tenant you wish to use.

> Create a new application in a second, new tenant. Because you need to know the `tenantId`, generate a new UUID using the `#{UUID()}` function call and assign that value to `secondTenantId`. Now you can re-use this value to create the tenant, and to make the Create Application API request.

Reviewer: ```suggestion
Create a new application in a second, new tenant. Because you need to know the `tenantId`, generate a new UUID using the `#{UUID()}` function call and assign that value to `secondTenantId`. Now, you can re-use this value to create the tenant and to make the Create Application API request.
```

include::docs/v1/tech/shared/_default-entities.adoc[]

> With the known Id, you can build other fields in your Kickstart file that depend on or copy them. For example, you may copy the default theme and modify the copied theme via Kickstart, or you may create a new tenant but set the tenant Id token signing key to one of the shadow keys.

Reviewer: The first sentence seems off to me but I am not sure why.  After reading several times, it makes sense, but I shouldn't have to re-read to understand.  This is just a suggestion.
```suggestion
With the known Ids, you can build other fields in your Kickstart file that utilizes them. For example, you may copy the default theme and modify the copied theme via Kickstart, or you may create a new tenant but set the tenant Id token signing key to one of the shadow keys.
```

```

> In this example you are creating an Email template and reading in the values for the text and html values from files in a subdirectory named `emails`. Using external files for templates like this allows you to format your emails nicely. Kickstart will handle the necessary JSON escaping to complete the API request.

Reviewer: I think it should be Email Template or email template.  I think the later makes the most sense.
```suggestion
In this example you are creating an email template and reading in the values for the text and html values from files in a subdirectory named `emails`. Using external files for templates like this allows you to format your emails nicely. Kickstart will handle the necessary JSON escaping to complete the API request.
```
